<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputPage</class>
 <widget class="QWidget" name="OutputPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>-1</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListView" name="outputs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>110</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32767</width>
       <height>110</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="progressListView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>104</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="showLogPushBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Show &amp;Log</string>
       </property>
       <property name="shortcut">
        <string>Alt+L</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="stopPushBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>S&amp;top</string>
       </property>
       <property name="shortcut">
        <string>Alt+T</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="startButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
 <tabstops>
  <tabstop>outputs</tabstop>
  <tabstop>showLogPushBtn</tabstop>
  <tabstop>stopPushBtn</tabstop>
 </tabstops>
 <includes>
  <include location="local">kpushbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>showLogPushBtn</sender>
   <signal>clicked()</signal>
   <receiver>OutputPage</receiver>
   <slot>showLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopPushBtn</sender>
   <signal>clicked()</signal>
   <receiver>OutputPage</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>OutputPage</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
